frontend/src/components/SepForm.js

```diff
@@ -16,19 +26,203 @@ const useStyles = makeStyles({
+                  disabled={participantId ? true : false}
+                >
+                  <InputLabel htmlFor="last_name">
+                    Participant&apos;s Last Name
+                  </InputLabel>
+                  <PrevPointInput
+                    id="last_name"
+                    name="last_name"
+                    onChange={handleChange}
+                    value={values.last_name}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12}>
+                <FormControl
+                  error={errors.date_of_birth && touched.date_of_birth}
+                  disabled={participantId ? true : false}
+                >
+                  <InputLabel shrink htmlFor="date_of_birth">
+                    Date of Birth
+                  </InputLabel>
+                  <PrevPointInput
+                    id="date_of_birth"
+                    name="date_of_birth"
+                    type="date"
+                    onChange={handleChange}
+                    value={values.date_of_birth}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12}>
+                <FormControl
+                  error={errors.mothers_last_name && touched.mothers_last_name}
+                  disabled={participantId ? true : false}
+                >
+                  <InputLabel htmlFor="mothers_last_name">
+                    Mother&apos;s Last Name
+                  </InputLabel>
+                  <PrevPointInput
+                    id="mothers_last_name"
+                    name="mothers_last_name"
+                    onChange={handleChange}
+                    value={values.mothers_last_name}
+                  />
+                </FormControl>
+              </Grid>
+              <Grid item xs={12}>
+                <PrevPointButton
+                  type="submit"
+                  disabled={(participantId ? true : false) || isSubmitting}
+                >
+                  Search
+                </PrevPointButton>
+              </Grid>
+            </Grid>
+          </Form>
+        )}
+      </Formik>
+      <Formik className={classes.form}>
+        <Form>
+          <Grid container>
+            <Grid item xs={12}>
+              <Grid container spacing={2}>
+                <Grid item xs={6}>
+                  <FormControl disabled={participantId ? false : true}>
+                    <InputLabel>Needles In</InputLabel>
+                    <Select name="needles_in" value="">
+                      {[...Array(20)].map((x, i) => (
+                        <MenuItem key={i + 1} value={i + 1}>
+                          {i + 1}
+                        </MenuItem>
+                      ))}
+                    </Select>
+                  </FormControl>
+                </Grid>
+                <Grid item xs={6}>
+                  <FormControl disabled={participantId ? false : true}>
```

Comment:
this can just be `disabled={!participantId}` instead of ternary

Reply:
Actually, this will throw an invalid prop console warning. The MUI component wants a boolean value.